uiowa/uiowa
# [Facilities] Include both current and upcoming alerts

Request from Facilities

> Before we migrated facilities.uiowa.edu to SiteNow v3, we had a previous iteration of this custom Alerts app that allowed us to publish construction alerts ahead of time, especially for high impact access alerts like this road closure on Madison Street for new student move-in. In our training for the system, we advise our construction managers to give as much advance notice as possible for access alerts so those commuting to or around campus are aware of closures ahead of time and can plan accordingly.  

> Can we adjust this custom app to allow us to include alerts in the Alerts block as soon as they are marked “published?”

> We would prefer to have the items sort by most recently published with the newest items at the top, and keep that “sticky” option if we were to have a super high-impact project that we want to display first.

# How to test


`ddev blt ds --site=facilities.uiowa.edu`

Check for a clean sync
Visit https://facilities.uiowa.ddev.site/alerts and see that the "S. Madison Street south of Washington Street between the Main Library and Seamans Center - Library and EPB Steam Service" alert appears at the top of the list, and that further alerts follow in descending order by start date.

Code review and or sync ITS and check that its alert lists block still only contains current (not upcoming) alerts.
